packages/core/src/base/Time.ts

```diff
@@ -19,68 +38,78 @@ export class Time {
   }
 
   /**
-   * Constructor of the Time.
+   * The delta time in seconds from the last frame to the current frame.
    */
-  constructor() {
-    this._clock = performance ? performance : Date;
-
-    this._timeScale = 1.0;
-    this._deltaTime = 0.0001;
-
-    const now = this._clock.now();
-    this._startTime = now;
-    this._lastTickTime = now;
+  get deltaTime(): number {
+    return this._deltaTime;
   }
 
-  reset() {
-    this._lastTickTime = this._clock.now();
+  /**
+   * The unscaled delta time in seconds from the last frame to the current frame.
+   */
+  get unscaledDeltaTime(): number {
+    return this._unscaledDeltaTime;
   }
 
   /**
-   * Current Time
+   * The amount of elapsed time in seconds since the start of the engine.
    */
-  get nowTime(): number {
-    return this._clock.now();
+  get elapsedTime(): number {
+    return this._elapsedTime;
   }
 
   /**
-   * Time between two ticks
+   * The amount of unscaled elapsed time in seconds since the start of the engine.
    */
-  get deltaTime(): number {
-    return this._deltaTime;
+  get unscaledElapsedTime(): number {
+    return this._unscaledElapsedTime;
   }
 
   /**
-   * Scaled delta time.
+   * Constructor of the Time.
    */
-  get timeScale(): number {
-    return this._timeScale;
-  }
-  set timeScale(s) {
-    this._timeScale = s;
+  constructor() {
+    this._lastSystemTime = performance.now() / 1000;
   }
 
   /**
-   * Unscaled delta time.
+   * @internal
    */
-  get unscaledDeltaTime(): number {
-    return this._deltaTime / this._timeScale;
+  _reset() {
+    this._lastSystemTime = performance.now() / 1000;
   }
 
   /**
-   * The elapsed time, after the clock is initialized.
+   * @internal
    */
-  get timeSinceStartup(): number {
-    return this.nowTime - this._startTime;
+  _update(): void {
+    const currentSystemTime = performance.now() / 1000;
+
+    const unscaledDeltaTime = currentSystemTime - this._lastSystemTime;
+    this._unscaledDeltaTime = unscaledDeltaTime;
```

Comment:
Why not `Math.min(unscaledDeltaTime, this.maximumDeltaTime)` here

Reply:
We have two version of "deltaTime": unscale version is `unscaledDeltaTime`, scale version is `deltaTime`, this is only scale version, maybe i should add another property named `unscaledMaximumDeltaTime` later?